pkg/cmd/server/api/types.go

```diff
@@ -154,17 +158,24 @@ type EtcdConfig struct {
 
 	PeerAddress   string
 	MasterAddress string
-	StorageDir    string
+	// StorageDir indicates where to save the etcd data
+	StorageDir string
 }
 
 type KubernetesMasterConfig struct {
-	MasterIP            string
-	ServicesSubnet      string
-	StaticNodeNames     []string
+	// MasterIP is the public IP address of kubernetes stuff.  If emtpy, the first result from net.InterfaceAddrs will be used.
+	MasterIP string
+	// ServicesSubnet is the subnet to use for assigning service IPs
+	ServicesSubnet string
+	// StaticNodeNames is the list of nodes that are statically known
+	StaticNodeNames []string
+	// SchedulerConfigFile points to a file that describes how to set up the scheduler.  If empty, you get the default scheduling rules.
 	SchedulerConfigFile string
 }
 
 type CertInfo struct {
+	// CertFile is a file holding the public half of public/private keypair
```

Comment:
don't mention keypairs... I don't think that's correct. "CertFile is a file containing a PEM-encoded certificate"